pkgs/development/compilers/julia/generic.nix

```
@@ -81,12 +102,16 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  preInstall = ''
+    export JULIA_SSL_CA_ROOTS_PATH=""
+  '';
```

```suggestion
  env.JULIA_SSL_CA_ROOTS_PATH = "";
```
Please add a comment about the reason behind.